index.html

```diff
@@ -0,0 +1,304 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Twitter Challenge</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!-- Header Section  -->
+
+    <header>
+      <div class="header-grid">
+        <div class="twitter-logo">
+          <img src="/Pictures/icons/Twitter logo.png" alt="" />
```

Comment:
There are alt attributes on all the images apart from this one, I think that's great.